plugins/GSdx/GSDevice.cpp

```diff
@@ -118,6 +144,19 @@ void GSDevice::Present(const GSVector4i& r, int shader)
 			ShaderConvert_COMPLEX_FILTER}; // FIXME
 
 		Present(m_current, m_backbuffer, GSVector4(r), s_shader[shader]);
+
+static std::chrono::system_clock::time_point timer;
```

Comment:
And maybe you could put an ifdef ENABLE_OSD_TEST (here and above). Then add the define in config.h